model/task/task_test.go

```diff
@@ -1253,6 +1253,43 @@ func TestBulkInsert(t *testing.T) {
 	}
 }
 
+func TestGetTaskIdBetweenIds(t *testing.T) {
+	assert := assert.New(t)
+	require.NoError(t, db.ClearCollections(Collection))
```

Comment:
nit: should be consistent about using a `assert.New` for `require.New` as well, or otherwise omit the `assert.New` entirely.